src/ManTooltip/widget/ManTooltip.js

```diff
@@ -96,15 +112,15 @@ mendix.widget.declare('ManTooltip.widget.ManTooltip', {
 				this.windowEvt = this.connect(document.body, 'onclick', this.windowClick);
 
 			if (this.position == 'popup') {
-				var coords = dojo.coords(this.imgNode, true);
-				dojo.style(this.helpNode, {
+				var coords = dojoHtml.coords(this.imgNode, true);

```

Comment:
in last widget you use dom-geometry position. Why do you use dojoHtml.coords this time?